glide.lock

```diff
@@ -65,7 +67,7 @@ imports:
   - context
   - publicsuffix
 - name: golang.org/x/sys
-  version: 7a6e5648d140666db5d920909e082ca00a87ba2c
+  version: 99f16d856c9836c42d24e7ab64ea72916925fa97
   subpackages:
   - unix
 testImports: []
```

Comment:
I guess it kind of should be in the repo then. Just makes a lot of commit noise.